.circleci/config.yml

```diff
@@ -544,6 +546,12 @@ gke_test: &gke_test
         command: |
           ./script/start-gke-env.sh $ESCAPED_GKE_CLUSTER $GKE_ZONE $GKE_BRANCH $GKE_ADMIN > /dev/null
     - <<: *install_helm_cli
+    - run:
+        # TODO(castelblanque) Unify shared resources with kubeapps-local-dev-users-rbac.yaml
+        # that only applies to Kind clusters
+        name: Apply customizations to GKE cluster
+        command: |
+          kubectl create namespace kubeapps-user-namespace
```

Comment:
This applies only to GKE. Needed now that we are making use of new rolebindings (see `script/e2e-test.sh`).